dropwizard-api/src/main/java/org/coner/api/entity/Registration.java

```diff
@@ -1,13 +1,24 @@
 package org.coner.api.entity;
 
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import org.hibernate.validator.constraints.NotBlank;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Null;
+
 /**
  * REST API entity representing a Registration
  */
+@JsonPropertyOrder({"id", "firstName", "lastName", "event"})
```

Comment:
I just followed what you had in Event.  I'm not against ordering the json properties, but I didn't think it was necessary to maintain a tight coupling of order between our fixtures and responses and hence the addition of the JSON assert that is order agnostic.